livre/eq/zeq/communicator.cpp

```diff
@@ -259,7 +259,7 @@ class Communicator::Impl
         // Receiving all queued events from all receivers without blocking.
         for( auto subscriber : subscribers )
             while( subscriber->receive( 0 ))
-                /*nop*/ ;
+                _config.sendEvent( REDRAW );
```

Comment:
They were needed for all events already. The Zerobuf-based event communication makes it easier to centralize this here, since the application is not involved anymore (no deserialization handler needed).